packages/volto/theme/themes/pastanaga/extras/widgets.less

```
@@ -280,3 +280,15 @@ body.babel-view .field.language-independent-field {
     }
   }
 }
+
+// ### FormFieldWrapper ###
+.language-independent-field label {
```

@iRohitSingh Careful -- this would also "leak" to any `label` tags inside the widget for a language-independent field. (That is probably uncommon, but I would guess there is one somewhere, since it's common to have custom widgets.)

I'd suggest making the selector a bit more specific:

```suggestion
.language-independent-field .wrapper > label {
```

---

Done, Thanks.